### PR TITLE
test(multiline): edge case tests for issue #18

### LINKED ISSUE #18: Tests: multiline edge cases
Depends on #12 and ideally #14 (basic outbound first).

- Empty lines preserved (consecutive `\n`s round-trip).
- Trailing newline preserved.
- Message exactly at the multiline byte boundary.
- Message exceeding ergo's `max-lines` falls back to legacy chunker (or whatever the current behavior is — assert what we want here).
- Mix of long logical lines (split into wire chunks with `draft/multiline-concat`) plus short ones.

### PR DESCRIPTION
**Pull request by TeakBuilds**
closes #18

## what

`src/constants.ts` — extracts `MULTILINE_LINE_BYTES` so tests don't hardcode 300.

`test/helpers/ergo.ts` — exports `ERGO_MAX_LINES` (matches `makeErgoConfig`).

`test/multiline-edge-cases.test.ts` — 6 tests covering all cases from the issue:
- empty lines preserved (`hello\n\nworld` round-trips)
- trailing newline preserved (`hello\nworld\n` round-trips)
- exactly at byte boundary → fast path (no batch overhead)
- one byte over → `draft/multiline batch` + concat; receiver reassembles
- 201 lines (> ergo max 200) → falls back to legacy chunker, no error
- mix of long+short logical lines → reassembles to exact original

all 35 tests pass.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by AlexSc** [comment]
lead-pm, can you file that followup issue please?

File: test/multiline-edge-cases.test.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    expect(result.isError).toBeFalsy()
+    expect(toolText(result)).toContain('draft/multiline batch')
+
+    const n = await receiver.waitForNotification(
+      n => n.meta.channel === '#ml-ec4' && n.meta.sender === 'ml-ec4-s',
+    )
+    expect(n.content).toBe(text)
+  })
+
+  it('exceeding max-lines falls back to legacy chunker', async () => {
```
Comment (AlexSc) [inline test/multiline-edge-cases.test.ts]: This is overspecifying. We should assert that exceeding max-lines still works. _HOW_ it works is out of scope for the test. Make sense?

Comment (AlexSc) [reply test/multiline-edge-cases.test.ts]: By which I mean "Assert you got the text you sent"

File: test/helpers/ergo.ts [hunk 2]
```diff
@@ -7,6 +7,9 @@ import { afterAll } from 'bun:test'
 
 const ROOST_ROOT = join(import.meta.dirname, '..', '..')
 
+/** Matches `max-lines` in makeErgoConfig below. */
+export const ERGO_MAX_LINES = 200
```
Comment (AlexSc) [inline test/helpers/ergo.ts]: This doesn't match maxLines from irc-server.ts. Not entirely sure what to do about that. I'm open to ideas on how to handle those constants in irc-server.ts

**Review by AlexSc — CHANGES_REQUESTED** [review]

**Review by AlexSc — APPROVED** [review]
I think I'd actually like an issue to drop legacy mode
